tests/testparse.cpp

```diff
@@ -378,6 +379,15 @@ void TestParse::exportCSVDiveDetails()
 	clear_dive_file_data();
 
 	parseCSVmanual(1, "testcsvexportmanualimperial.csv");
+
+	// We do not currently support reading SAC, thus faking it
+	if (dive_table.nr > 0) {
+		struct dive *dive = dive_table.dives[dive_table.nr - 1];
+		dive->sac = 1049;
+	}
```

Comment:
wouldn't `process_loaded_dives()` calculate this for you?

Reply:
Nope, we don't have the samples anymore. But I'll just save the SAC before clearing original data.